modules/heat_conduction/src/interfacekernels/HeatConvectionInterface.C

```diff
@@ -0,0 +1,62 @@
+#include "HeatConvectionInterface.h"
+
+#include "metaphysicl/raw_type.h"
+
+using MetaPhysicL::raw_value;
+
+registerMooseObject("HeatConductionApp", HeatConvectionInterface);
+
+template <>
+InputParameters validParams<HeatConvectionInterface>()
+{
+  InputParameters params = validParams<InterfaceKernel>();
+  params.addRequiredParam<MaterialPropertyName>("htc", "heat transfer coefficient");
+  return params;
+}
+
+HeatConvectionInterface::HeatConvectionInterface(const InputParameters & parameters)
+  : InterfaceKernel(parameters),
+    _h(getADMaterialPropertyByName<Real>(this->template getParam<MaterialPropertyName>("htc")))
+{
+  if (!parameters.isParamValid("boundary"))
```

Comment:
Looks like this comes from Pronghorn :) We actually don't need this error here anymore, `InterfaceKernel` does the same check. Not sure why it was ever here in the first place.

Reply:
It definitely does. :) I was wondering myself if this was even something we needed. Thanks!